File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_DM_TenDanhHieuThiDua.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_DM_TenDanhHieuThiDua } from './NS_DM_TenDanhHieuThiDua.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_DM_TenDanhHieuThiDuaService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Ma', 'TenDanhHieu', 'Diem', 'GhiChu'];

    List: NS_DM_TenDanhHieuThiDua[] | undefined;
    ListFilter: NS_DM_TenDanhHieuThiDua[] | undefined;
    FormData!: NS_DM_TenDanhHieuThiDua;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_DM_TenDanhHieuThiDua";
    }
}

File: Service_eHospital_DongNai_A_NSTL/Implement/NS_DM_TenDanhHieuThiDuaService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_DM_TenDanhHieuThiDuaService : BaseService<NS_DM_TenDanhHieuThiDua, INS_DM_TenDanhHieuThiDuaRepository>
    , INS_DM_TenDanhHieuThiDuaService
    {
    private readonly INS_DM_TenDanhHieuThiDuaRepository _NS_DM_TenDanhHieuThiDuaRepository;
    public NS_DM_TenDanhHieuThiDuaService(INS_DM_TenDanhHieuThiDuaRepository NS_DM_TenDanhHieuThiDuaRepository) : base(NS_DM_TenDanhHieuThiDuaRepository)
    {
    _NS_DM_TenDanhHieuThiDuaRepository = NS_DM_TenDanhHieuThiDuaRepository;
    }
    }
    }

